lib/Event.pm

```diff
@@ -204,9 +204,9 @@ _load_watcher($_) for qw(idle io signal timer var);
 
 # Provide hints to Inline.pm for usage:
 # use Inline with => 'Event';
+# note: return information for all values of $language
 sub Inline {
-    my ($class, $language) = @_;
-    return if $language ne 'C'; # Inline gives good error message
```

Comment:
Please only delete this line, the note doesn't add value, nor does the change to the params line.

Reply:
will fix